test/request.js

```diff
@@ -19,10 +19,10 @@ var assert = require('assert');
 var nock = require('nock');
 var request = require('../includes/request.js');
 
-describe('#unit Check request response', function() {
-  const url = 'http://localhost:5984';
-  const client = request.client(url, 1);
+const url = 'http://localhost:5984';
```

Comment:
Will running nock on port 5984 be a problem if we ever want to run in an environment with a local couch running?